esp/src/eclwatch/Utility.js

```diff
@@ -19,6 +20,13 @@
         ;
     }
 
+    function csvEncode(str) {
+        str = "" + str;
+        return str.replace(/"/g, '\"')
+                  .replace(/','/g, '\'')
+        ;
+    }
```

Comment:
Technically strings only need to be placed in quotes if the conatain
* "
* \n
* \r
* ,

But the spec allows quotes on all strings and its quicker to add quotes than to check for those symbols.